testing/cachefilesd-inotify/APKBUILD

```diff
@@ -0,0 +1,35 @@
+# Contributor: Jake Buchholz <tomalok@gmail.com>
+# Maintainer: Jake Buchholz <tomalok@gmail.com>
+pkgname=cachefilesd-inotify
+pkgver=0.11.0
+pkgrel=0
+pkgdesc="Userspace daemon acting as a backend for FS-Cache (uses INOTIFY instead of DNOTIFY)"
+url="https://gitlab.com/tomalok/cachefilesd-inotify"
+arch="all"
+license="GPL"
+depends="attr"
+makedepends="rpm file"
+replaces="cachefilesd"
+options="!check"
+subpackages="$pkgname-doc"
+source="https://gitlab.com/tomalok/cachefilesd-inotify/-/archive/$pkgver/$pkgname-$pkgver.tar.bz2
+	cachefilesd.initd"
+builddir="$srcdir"/$pkgname-$pkgver
+
+build() {
+	cd "$builddir"
```

Comment:
No need for this, abuild does this by default.

Reply:
ok.